docs/development/code_guide.rst

```diff
@@ -5,14 +5,73 @@ Code Development Guidelines
 This document describes the coding requirements and guidelines to be
 followed during the development of PlasmaPy and affiliated packages.
 
-Dependencies
-============
+Code written for PlasmaPy must be compatible with Python 3.6 and
+later. Python 2 is not supported by PlasmaPy.
+
+PlasmaPy requires 
+
+* Astropy 2.0 or later
+* NumPy 1.13 or later
+* scipy 0.19 or later
+* matplotlib 2.0 or later
+
+Setting up an environment for development
+=========================================
+
+To obtain PlasmaPy's source code, use the usual
+
+.. code-block:: bash
+
+  git clone https://github.com/PlasmaPy/PlasmaPy.git 
+
+To be able to import PlasmaPy from your source version:
+
+.. code-block:: bash
+
+  cd PlasmaPy
+  python setup.py develop
```

Comment:
You should avoid using `python setup.py develop` in favor of `pip install -e ./` using `develop` can lead to having (many) multiple copies of the library installed at once.

Reply:
huh, today I learned! Thanks a bunch!